src/Ajax_Inspector/Plugin.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Plugin class file
+ */
+
+namespace SDOKUS\Ajax_Inspector;
+
+// Make sure that the file is being accessed in the WP environment.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Make sure that the class is not being duplicated.
+if ( ! class_exists( 'Dokus__Ajax__Inspector' ) ) {
+	/**
+	 * The Ajax Inspector Class
+	 *
+	 * This is the main class of the plugin
+	 */
+	class Dokus__Ajax__Inspector {
```

Comment:
We are inside of the namespace, so we should call this something like `Plugin`. Making the fully qualified name be something like `\Sdokus\Ajax_Inspector\Plugin`.